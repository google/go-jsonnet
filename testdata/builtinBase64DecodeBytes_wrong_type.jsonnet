std.base64DecodeBytes(1)