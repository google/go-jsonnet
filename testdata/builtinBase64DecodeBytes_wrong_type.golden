RUNTIME ERROR: base64DecodeBytes requires a string, got number
-------------------------------------------------
	testdata/builtinBase64DecodeBytes_wrong_type:1:1-25	$

std.base64DecodeBytes(1)

-------------------------------------------------
	During evaluation	


